.env.example

```
@@ -32,3 +32,8 @@ GOOGLE_CLIENT_SECRET=
 ITEMS_SERVICE_PORT=
 ITEMS_SERVICE_HOST=dogapi.dog
 ITEMS_SERVICE_API_KEY=
+
+# Players Service
+PLAYERS_SERVICE_PORT=
```

Sugerimos que se deje seteado un puerto para estandarizar el servicio de jugadores:
```suggestion
PLAYERS_SERVICE_PORT=8010
```

---

Me parece que esto debería ir en otra US. Actualmente el puerto seteado en el **players-service** en el [.env.example](https://github.com/TPII-PadelPals/players-service/blob/dev/.env.example#L17) es el `8000` al igual que en todos los servicios creo. Si se estandariza de alguna manera habria que hacerlo de una para todos asi se sigue la misma convención.

---

Se crea US: https://tpii-padel-pals.atlassian.net/browse/SCRUM-256